File: C#/Mathematics/count-digits.cs
// C# program to print Hello World!
using System;

// namespace declaration
namespace CountDigitsClass {
	
	// Class declaration
	class CountDigits {
		
        public int count(int x) {
            int res = 0;
            while(x > 0) {
                x = x/10;
                res++;
            }
            return res;
        }

		// Main Method
		static void Main(string[] args) {
		    CountDigits countDigits = new CountDigits(); // Creating Object  
			Console.WriteLine(countDigits.count(123));
		}
	}
}

File: C++/Mathematics/count-digits.cpp
#include <iostream>

using namespace std;

int CountDigits(int x) {
    int res = 0;
    while(x > 0) {
        x = x/10;
        res++;
    }
    return res;
}

int main() {
    cout << CountDigits(12345);    
}